scripts/google-free.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+sed -i -e '$aGOOGLE_FREE=1' .env.release
+
+#------------------------------------
+# Remove Android Remote Notifications
```

Comment:
This comment isn't very useful. It just describes what happens bellow.
Comments should describe __why__ something is happening, not __what__.
Same goes for the line above, it would be nice to add a comment like:
```sh
# Gradle parameter that makes some dependencies optional.
sed -i -e '$aGOOGLE_FREE=1' .env.releases
```
This makes it clear why the change is made, not what is being done, which is clear from the command itself.

Reply:
Actually, that comment isn't fully accurate, because it is also used in Clojure.